tests/unit/transaction-details.spec.js

```diff
@@ -113,7 +113,6 @@ function mountComponent({ hasError = false } = {}) {
       getTx: () => () => testTransaction,
       getTxType: () => () => 'provide liquidity',
       getTxSymbol: () => () => AE_SYMBOL,
-      getTransactionTipUrl: () => () => '',
```

Comment:
Why?

Reply:
Because `getTransactionTipUrl` is not anymore present in the store so there is no need to mock it.